hedyorg/hedy
# 💻 teachers feedback option

Fixes #5205

**How to test?**
- go to any page,
- you should see the feedback button in the bottom-right corner. 
- Click it, write a message and select category.
- send it
